src/app/utils/leading-zeros.ts

```diff
@@ -0,0 +1,9 @@
+export function leadingZeros(value: number, length: number) {
```

Comment:
is it used anywhere?

Reply:
removed in next commit. please re-review :)